package.json

```diff
@@ -16,6 +16,7 @@
     "validate-dev": "npm run lint && npm run build && npm outdated && npm run test-dev",
     "watch:build": "watch --wait=5 'npm run validate-dev' ./app/",
     "dev": "npm run watch:build",
+    "healthcheck": "node app/healthcheck.js",
```

Comment:
We should probably call this `start-healthcheck` and we should add `npm run start-healthcheck` to the `start` script, as well.